runtime/wasmer_test.go

```diff
@@ -613,6 +646,61 @@ func TestExt_blake2_256_enumerated_trie_root(t *testing.T) {
 	}
 }
 
+// test that ext_twox_64 performs a xxHash64
+func TestExt_twox_64(t *testing.T) {
+	runtime, err := newTestRuntime()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	mem := runtime.vm.Memory.Data()
+	// save data in memory
+	// test for empty []byte
+	data := []byte(nil)
+	pos := 170
+	out := pos + len(data)
+	copy(mem[pos:pos+len(data)], data)
+
+	// call wasm function
+	testFunc, ok := runtime.vm.Exports["test_ext_twox_64"]
+	if !ok {
+		t.Fatal("could not find exported function")
+	}
+
+	_, err = testFunc(pos, len(data), out)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	//check result against expected value
+	t.Logf("Ext_twox_64 data: %s, result: %s", data, hex.EncodeToString(mem[out:out+8]))
```

Comment:
Not sure why we would log this, would make more sense in case of failure otherwise this info is irrelevant

Reply:
removed